client/views/contact/contact.index.html

```diff
@@ -18,13 +18,13 @@
                     <td><input class="form-control" ng-model="contactCtrl.formData.number"/></td>
                     <td>
                         <button type="submit" class="btn btn-success">
-                            <span class="fa fa-plus"></span>
+                            <span ng-class="contactCtrl.updateIcon || 'fa fa-plus fa-fw'"></span>
                         </button>
                     </td>
                     <td ng-show="contactCtrl.isEditing">
                         <!-- type="button" required for html to not treat it as a submit button -->
                         <button type="button" class="btn btn-danger" ng-click="contactCtrl.stopEditing()">
-                            <span class="fa fa-times"></span>
+                            <span class="fa fa-times fa-fw"></span>
```

Comment:
I forgot to mention in the summary that I also changed all the font-awesome icons to be fixed-width for consistency. Otherwise, the size of the button was changing.